.env.local.example

```
@@ -8,4 +8,6 @@ NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=
 NEXT_PUBLIC_FIREBASE_APP_ID=
 NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=
 NEXT_PUBLIC_FIREBASE_COLLECTION_NAME=
-NEXT_PUBLIC_FIREBASE_COLLECTION_NAME=
\ No newline at end of file
+NEXT_PUBLIC_FIREBASE_COLLECTION_NAME=
+CHATGENIE_EMAIL=
+CHATGENIE_PASSWORD=
```

```suggestion
NEXT_PUBLIC_CHATGENIE_EMAIL=
NEXT_PUBLIC_CHATGENIE_PASSWORD=
```
NEXT_PUBLIC_ Mambi bo'masa vrode ishlamidi @KamronBegmatov